src/utils/api.ts

```diff
@@ -120,6 +120,12 @@ export const getReviews = (
   locationId: paths['/locations/{id}/reviews']['get']['parameters']['path']['id'],
 ) => handleResponse(instance.get(`/locations/${locationId}/reviews`))
 
+export const getImports = () => handleResponse(instance.get(`/imports`))
+
+export const getImportById = (
+  id: paths['/imports/{id}']['get']['parameters']['path']['id'],
```

Comment:
It's not immediately clear to me what this does (maybe requires context on the rest of the app though)

Reply:
This specifies the type of id from the generated TypeScript API schema.